### PR TITLE
Adds unum

### PR DESCRIPTION
**Pull request by utilbird**
## About The Pull Request

Adds unum, a card game "based off" uno

## Why It's Good For The Game

I mean it's uno, who doesn't want uno

## Changelog
:cl:
add: adds unum
/:cl:

### CODE REVIEW

**Comment by utilbird** [comment]
oops, forgot the icon lmao

**Comment by Trilbyspaceclone** [comment]
Poor pool noodle

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -832,16 +833,16 @@
...
+
+/obj/item/toy/cards/deck/update_icon()
+	if(cards.len > original_size/2)
+		icon_state = "deck_[deckstyle]_full"
+	else if(cards.len > original_size/4)
+		icon_state = "deck_[deckstyle]_half"
+	else if(cards.len > 0)
+		icon_state = "deck_[deckstyle]_low"
+	else if(cards.len == 0)
+		icon_state = "deck_[deckstyle]_empty"
```
Comment (Ghommie) [inline code/game/objects/items/toys.dm]:
```suggestion
/obj/item/toy/cards/deck/update_icon_state()
	switch(cards.len)
		if(INFINITY to original_size/2)
			icon_state = "deck_[deckstyle]_full"
		if(original_size/2 to  original_size/4)
			icon_state = "deck_[deckstyle]_half"
		if(original_size/4 to 1)
			icon_state = "deck_[deckstyle]_low"
		else
			icon_state = "deck_[deckstyle]_empty"
```
That's some regression we have to fix in here.

**Review by Ghommie — CHANGES_REQUESTED** [review]
Finally, now I can unum reverse card people's statements.

By the by, it seems you are removing the pool noodle state and modifying the virologist figurine, place take care to fix that thank you.

File: code/game/objects/items/toys.dm (hunk 1 above)
Comment (utilbird) [reply code/game/objects/items/toys.dm]: Ah you’re right. I’ll fix this when I get home

**Review by Ghommie — APPROVED** [review]
Just noticed you also duplicate states of cards of the suit of hearts. Could you please delete said duplicates, pretty please? Overall lgtm.

**Comment by utilbird** [comment]
ah, sorry

**Pull request merged**